src/index.js

```diff
@@ -0,0 +1,28 @@
+import { createTransform } from "redux-persist";
+import LZ from "lz-string";
+import stringify from "json-stringify-safe";
+import process from "process";
```

Comment:
I think this is unnecessary. Afaict `process` module is intended to provide nextTick functionality, process.env already exists.